### ISSUE TITLE
Blood (1997)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#2000]
I have found the problem with 3dfx Blood that causes it to chug and therefore a solution. The issue is caused by rendering the shadows, so by turning the 3dfx shadows off we have a smooth game. In 3dfx.bat there is a setting "set BUILD_NOFOG=0", set this to "set BUILD_NOFOG=1" to switch off the shadows. It sucks that the shadows cause the issue, but in my opinion better to have a playable 3dfx version and make a specific issue in github noting what weve done for future reference than to have an at times unplayable game.

**Comment by exoscoriae** [comment]
Added option to launch 3dfx w/ or w/out shadows, with a note about performance.

**Comment by exoscoriae** [comment]
@Python-Exoproject Looking over this: https://github.com/dosbox-staging/dosbox-staging/wiki/Game-issues#Blood
I believe we have not addressed the save game damage scaling issue.

However, we don't just have a single Blood release. We have Crypt, Plasma, etc... And it is unclear to me if this bug persists in those games as well.

I'm trying to think on how we can test this....

**Comment by exoscoriae** [comment]
seems we have multiple blood tickets currently.

**Comment by exoscoriae** [comment]
I hadn't included this previously because it was reported to cause issues.

Now that there is a v2, maybe it is better....

Seems really wonky to me though. I went ahea dnad backed up the old blood.exe to _blood.exe and added this one.

as far as the other versions go, iirc, it was a known issue at the time. but instead of an offiical patch, they just released the add on packs to address it.

**Comment by Python-Exoproject** [comment]
Did the new Blood exe work for you? I get the screen image below when I try it. Im pretty certain its a version mismatch as the new exe shows as v1.21 in the initial load screen while our original blood.exe shows v1.11



![Image](https://github.com/user-attachments/assets/fa82ba16-08ff-46d4-8b16-c6a35ca90ce0)

**Comment by exoscoriae** [comment]
well... this goes back to what I said. I hadn't included it because it was reported to cause issues.

I've reverted it.

**Comment by exoscoriae** [comment]
the issue *does* exist in our base blood game currently
it also exists in cryptic passages
it also exists in plasma pak

I tested all three using the dynamite jump technique in the first level of all three.

**Comment by exoscoriae** [comment]
The patch is specifically for "one whole unit of blood". Not the original DOS versiosn that we have. As there was no official "1.21" patch. The version we have is the latest patch they released.

I went and downloaded one whole unit and installed it for testing. The new blood.exe launched and the damage was fixed. But, the game had graphical issues (even without the patched blood executable). 
<img width="1259" alt="blood" src="https://github.com/user-attachments/assets/e73c60cb-b2e7-4a37-ad51-56e5faf9ae13">


At this point, to use the fixed exe would require using a version of the game that appears unstable.
